content/03.methods.md

```diff
@@ -27,6 +27,13 @@ For experiments where "compressed" methylation data was used, we used principal
 We initially applied the beta-mixture quantile normalization (BMIQ) method [@doi:10.1093/bioinformatics/bts680] to correct for variability in signal intensity between type I and type II probes, but we observed that this had no effect on our results.
 We report uncorrected results in the main paper for simplicity.
 
+### Cancer gene data download and integration
+
+To get a comprehensive picture of classification performance across a wide variety of cancer-related genes, we integrated several curated gene sets from the literature into a single "merged" cancer gene set.
+The individual gene sets we integrated were from Vogelstein et al. 2013 [@doi:10.1126/science.1235122] (all genes from Table S2A), Bailey et al. 2018 [@doi:10.1016/j.cell.2018.02.060] (only genes annotated as "pan-cancer" drivers in Table S1), and the COSMIC Cancer Gene Census [@doi:10.1038/s41568-018-0060-1] (all Tier 1 genes annotated as "somatic").
```

Comment:
```suggestion
The individual gene sets we integrated were from Vogelstein et al. [@doi:10.1126/science.1235122] (all genes from Table S2A), Bailey et al. [@doi:10.1016/j.cell.2018.02.060] (only genes annotated as "pan-cancer" drivers in Table S1), and the COSMIC Cancer Gene Census [@doi:10.1038/s41568-018-0060-1] (all Tier 1 genes annotated as "somatic").
```
I think safe to remove the years unless journal style requires it since the citation is there